src/templates/base/store/global/constants.js

```diff
@@ -0,0 +1 @@
+export const DUMMY_CONST = 'DUMMY CONSTANT';
```

Comment:
constant should always have same value as its Key/Name

Reply:
done.